packages/compass-crud/electron/renderer/index.js

```diff
@@ -105,8 +105,6 @@ const dataService = new DataService(connection);
 appRegistry.emit('data-service-initialized', dataService);
 
 dataService.connect((error, ds) => {
-  if (!error) appRegistry.emit('compass:status:done');
```

Comment:
This is the only thing I have to double-check. It is the last thing outside compass status that emits an event for compass status. Which is just weird. If we really need it I think we should use `appRegistry.getAction('Status.Actions');` like we do everywhere else.